packages/front-end/components/Experiment/TabbedPage/Implementation.tsx

```diff
@@ -81,6 +82,19 @@ export default function Implementation({
             </Button>
           ) : null}
         </div>
+        {experiment.variations.length > 2 &&
+        experiment.type !== "multi-armed-bandit" ? (
+          <PremiumCallout
+            id="exp-implementation-bandit-promo"
+            commercialFeature="multi-armed-bandits"
+            docSection="bandits"
+            dismissable={true}
+            mx="3"
+            mb="5"
+          >
+            Bandits can help you quickly find the best performing variant.
+          </PremiumCallout>
+        ) : null}
```

Comment:
When they already have Pro, instead of linking to docs, we should have a CTA that converts the experiment to a bandit instead.  This action already exists in the More menu at the top of the experiment page, so it's just a matter of opening the same confirmation modal from here.

![image](https://github.com/user-attachments/assets/6f70383d-f809-469c-91e5-7b8251c02399)